astropy/io/votable/tree.py

```diff
@@ -3132,32 +3132,38 @@ def _parse_parquet(self, iterator, config):
                 NotImplementedError,
             )
 
-        # Hack to keep windows working
         try:
-            fd = urllib.request.urlopen(href)
-        except urllib.error.URLError:
             # Hack to keep windows working
```

Comment:
Just one time is probably enough - remove if you still need to iterate (otherwise, don't bother!)

Reply:
You mean flatten the nested `try`? I am not sure if that is doable given the `if` within the outer `try`. The goal of the outer try is to catch any flavor of `fd` and then attempt to close it after Table parsing. Suggestion is welcome!

Reply:
No, I just meant to remove the comment, since it is done another time. But let's ignore it!

Reply:
Keep the comment (duplicating was a squash leftover, but having it once is useful) as the whole except part was needed for windows only.